### DISCUSSION TITLE
3.4

### DISCUSSION THREAD

**Discussion by abarichello** [abarichello/godot-ci#66] [Releases]
Release 3.4

<hr /><em>This discussion was created from the release <a href='https://github.com/abarichello/godot-ci/releases/tag/3.4'>3.4</a>.</em>

**Comment by fuzuli-gulmammadli** [comment]
Can you update the 3.4 version to be the same as 3.4.4 version. Because I am having issues to export to android :(

**Reply by Calinou** [reply]
Can't you just update your local target to be 3.4.4 in your CI file? 3.4.4 should be deployed already: https://github.com/abarichello/godot-ci/pull/79

**Reply by fuzuli-gulmammadli** [reply]
I did but I am getting this error when I am exporting to Android:
ERROR: _fs_changed: Project export failed with error code 12 for preset 'android'.

**Reply by fuzuli-gulmammadli** [reply]
I tried to build my own image with 3.4 version. But I am getting error in getbutler.sh script.

**Reply by fuzuli-gulmammadli** [reply]
I managed to create my own image by just changing the godot version to 3.4. But still got the same error when exporting to Android. 

The command I am using is: 
godot --export-debug Android build/app.apk.

And error that i am getting is: 
ERROR: Project export failed with error code 12 for preset "Android".
   at: _fs_changed (editor/editor_node.cpp:801)

**Reply by fuzuli-gulmammadli** [reply]
I was able to solve the issue. It was due to the apksigner. I disabled the signing in export_presets.cfg by making package/signed=false. And now it exports without issue.
